compatibility/bazel_tools/testing.bzl

```diff
@@ -294,10 +294,10 @@ excluded_test_tool_tests = [
         ],
     },
```

Comment:
Are you sure this solves the issue? If you look at the logs parts of the failures where from
```
//:ledger-api-test-tool-0.0.0-platform-1.14.2-postgresql     
```
I don’t see why your change affects that version range.

Reply:
My changes fixes:
```
 //:ledger-api-test-tool-0.0.0-platform-1.16.0-snapshot.20210727.7476.0.b5e9d861                             \
 //:ledger-api-test-tool-0.0.0-platform-1.16.0-snapshot.20210727.7476.0.b5e9d861-postgresql
```
I did not realize that I broke the test in 2 different ways.